src/fiatExchanges/UniquePaymentSection.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import { CICOFlow, FetchProvidersOutput, PaymentMethod } from 'src/fiatExchanges/utils'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+
+export interface UniquePaymentSectionProps {
+  paymentMethod: PaymentMethod
+  uniqueProvider: FetchProvidersOutput | null
+  setNoPaymentMethods: React.Dispatch<React.SetStateAction<boolean>>
+  flow: CICOFlow
+}
+
+export function UniquePaymentSection({
```

Comment:
It seems like you're trying to create a component that can be re-used for more bespoke payment methods in the future (that aren't part of FiatConnect or covered by an existing method). Personally I see this as an unnecessary abstraction. We have no plans to support more bespoke payment methods, and until we do, it provides extra cognitive effort for little gain

Reply:
+1. I think this could just be a `CoinbasePaymentSection` component or something similarly named; no reason to have an abstraction here when this will only ever be used for Coinbase.

Reply:
Okay! Thanks for the clarification.